src/providers/translations/TranslationsProvider.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import React, {
+  useContext,
+} from 'react';
+import defaultTranslations from '../../translations/en';
+import { mergeDeep } from '../../utils/mergeDeep';
+import TranslationsContext from './TranslationsContext';
+
+const TranslationsProvider = ({
+  children,
+  translations,
+}) => {
+  const context = useContext(TranslationsContext);
+
+  return (
+    <TranslationsContext.Provider
+      value={mergeDeep(context?.translations, translations)}
+    >
+      {children}
+    </TranslationsContext.Provider>
+  );
+};
+
+TranslationsProvider.defaultProps = {
+  children: null,
+  translations: defaultTranslations,
```

Comment:
It is weird that `defaultTranslations` is used in `TranslationsContext` as default value. Then again here in `TranslationsProvider.defaultProps`. It was before so we might leave as it is, but it looks weird now.

I am just curious that `|| {}` disappeared from `mergeDeep(context?.translations || {}, translations)`. I can't remember the purpose, but are we sure it can be removed from here and `GlobalPropsProvider.jsx`?

Reply:
> It is weird that defaultTranslations is used in TranslationsContext as default value.

Default value is now `{}`

> but are we sure it can be removed from here and GlobalPropsProvider.jsx

Before the default value was `null`. I changed it to `{}` so now we can ommit the `|| {}` part. I can see no issue with it.